app/javascript/stylesheets/application.scss

```diff
@@ -140,3 +140,33 @@ $govuk-assets-path: "~govuk-frontend/govuk/assets/";
 .sub-list {
   line-height: 2;
 }
+
```

Comment:
A bit of custom css because the new design isn't exactly like any of the options offered by the govuk styling system.